docs/start/overview.md

```diff
@@ -8,21 +8,100 @@ activePill: overview
 
-Microservice Integration
+## Shield: Service Discovery 
 
-1. Service Discovery
-1. Endpoint Discovery (Swagger)
+To make the life of an infrastructure engineer as easy as possible, your tooling must be as automated as possible.  
+Shield integrates with existing service discovery mechanisms to detect new deployments and scaling events and automatically
+start sending traffic to them.
 
-Production Ready
+<img src="{{ site.baseurl }}/assets/images/service_discovery.gif" class="img-responsive">
+
+As these new hosts come online, Shield will retrieve their [Swagger Documentation](http://swagger.io) that describes 
+what endpoints they are capable of serving.  Shield will intelligently route traffic to that host if it's actually 
+capable of handling the request.
+
+## Shield: The Router
+
+When Shield is in front of multiple unique services, it needs to know where to forward the request on to.  To
+solve this routing problem, Shield builds a routing table for all the upstream endpoints that were discovered
+during the Service Discovery phase.  The routing table then maps a request to a service by looking at the HTTP method, 
+path, and content type of the request and desired response.
+
+<img src="{{ site.baseurl }}/assets/images/routing.gif" class="img-responsive">
+
+This routing is fast, on the order of 0.01ms, even when choosing between hundreds of endpoints exposed by the upstream
+services.  
+
+Shield is smart when multiple upstream endpoints could be used to satisfy the request, and uses the [most
+specific endpoint]({{ site.baseurl }}/start/lifecycle/endpoint_sorting) possible when deciding which upstream to proxy
+to.
+
+## Shield: The Load Balancer
+
+When Shield is in front of multiple instances of the same service, it will load balance requests across the different 
+instances.  This load balancing occurs on a per endpoint basis, so even if different services advertise the same 
+endpoint (eg, if you're refactoring something out to a new service) Shield will balance the requests.
+
+<img src="{{ site.baseurl }}/assets/images/load_balancing.gif" class="img-responsive">
+
+Shield has weighted load balancing logic, and will smoothly transition between weight values.  In the example above,
+"Service B" could be the next version of "Service A".  Updating the weight on Service B would smoothly transition the 
+load to the new deploy, giving it ample time to scale up if needed.  This is ideal for graceful blue/green deployments.
+
+
+## Shield: The Proxy
+
+Shield takes advantage of it's position as a proxy to help guard the upstream services during times of trouble.  Each 
+upstream service is guarded by a [Circuit Breaker](http://doc.akka.io/docs/akka/2.4.4/common/circuitbreaker.html) so that
+Shield will automatically re-route or shed traffic if the upstream service is experiencing consistent failures or slowdowns.  
+Additionally, each endpoint on the upstream service has it's own dedicated Circuit Breaker so that one endpoint that 
```

Comment:
The first "that" makes me read this sentence with an amusing intonation.

* "We all have _that one relative_ who thinks he can grill and ends up burning the hot dogs at the family reunion."
* "its own dedicated Circuit Breaker so _that one endpoint_ that relies on a flaky resource..."

I would eliminate the first "that": "so one endpoint that relies..."